### PR TITLE
Add Release Notes for iOS Release 1.61.1

### PR DESCRIPTION
**Pull request by Uni-verse**
## Summary of Changes

This pull request fixes #[8625](https://github.com/brave/brave-ios/issues/8625)

## Submitter Checklist:

- [ ] *Unit Tests* are updated to cover new or changed functionality
- [ ] User-facing strings use `NSLocalizableString()`
- [ ] New or updated UI has been tested across:
  - [ ] Light & dark mode
  - [ ] Different size classes (iPhone, landscape, iPad)
  - [ ] Different dynamic type sizes

## Test Plan:

N/A

## Reviewer Checklist:

- [ ] Issues include necessary QA labels:
  - `QA/(Yes|No)`
  - `bug` / `enhancement`
- [ ] Necessary [security reviews](https://github.com/brave/security/issues/new/choose) have taken place.
- [ ] Adequate unit test coverage exists to prevent regressions.
- [ ] Adequate test plan exists for QA to validate (if applicable).
- [ ] Issue and pull request is assigned to a milestone (should happen at merge time).

### CODE REVIEW

**Comment by Uni-verse** [comment]
@iccub Could you please add tag for 1.61.1 whenever you get a chance, thanks!

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## [1.61.1](https://github.com/brave/brave-ios/releases/tag/v1.61.1)
+
+- Fixed Ad notification click event not triggering on iOS. ([#8619](https://github.com/brave/brave-ios/issues/8619))
```
Comment (kjozwiak) [inline CHANGELOG.md]: I don't think you need to mention `iOS` as these notes are all related to `iOS`. So probably can use:

```
- Fixed Brave Ad notification click event not being triggered. ([#8619](https://github.com/brave/brave-ios/issues/8619))
```

Added `Brave` in front of `ad` just so users don't get confused about that `ad` were talking about.

**Review by kjozwiak — CHANGES_REQUESTED** [review]
Added a few nit/improvements that can be made.

**Review by iccub — APPROVED** [review]

**Review by kjozwiak — APPROVED** [review]
LGTM 👍

**Comment by Uni-verse** [comment]
> LGTM 👍

Appreciate the review!

**Pull request merged**